src/utils/UpdateNotification.js

```diff
@@ -389,6 +509,16 @@ define(function (require, exports, module) {
 
                         // Update prefs with the last notified build number
                         lastNotifiedBuildNumber = allUpdates[0].buildNumber;
+
+                        //If no checksum field is present then we're setting it to 0, just as a safety check,
+                        // although ideally this situation should never occur in releases post its introduction.
+                        var platformInfo = getPlatformInfo(),
```

Comment:
If you want to retain this , you might want to move all of this to `notifyGetUpdateInstaller`